lib/gems/pending/MiqVm/MiqVm.rb

```diff
@@ -83,18 +83,11 @@ def openDisks(diskFiles)
       if @ost.miqVim
         dInfo.vixDiskInfo            = {}
         dInfo.vixDiskInfo[:fileName] = @ost.miqVim.datastorePath(df)
-        if @ost.miqVimVm && @ost.miqVim.isVirtualCenter?
-          thumb_print    = VcenterThumbPrint.new(@ost.miqVimVm.invObj.server)
-          @vdlConnection = @ost.miqVimVm.vdlVcConnection(thumb_print) unless @vdlConnection
+        if @ost.miqVimVm
+          @vdlConnection ||= @ost.miqVimVm.vdlVcConnection
           $log.debug "openDisks (VC): using disk file path: #{dInfo.vixDiskInfo[:fileName]}"
-        elsif @ost.miqVimVm
-          esx_host       = @ost.miqVimVm.invObj.server
-          esx_username   = @ost.miqVimVm.invObj.username
-          esx_password   = @ost.miqVimVm.invObj.password
-          thumb_print    = ESXThumbPrint.new(esx_host, esx_username, esx_password)
-          @vdlConnection = @ost.miqVimVm.vdlVcConnection(thumb_print) unless @vdlConnection
         else
-          @vdlConnection = @ost.miqVim.vdlConnection unless @vdlConnection
+          @vdlConnection ||= @ost.miqVim.vdlConnection
           $log.debug "openDisks (ESX): using disk file path: #{dInfo.vixDiskInfo[:fileName]}"
```

Comment:
How about we just print `"openDisks: using disk file path: #{dInfo.vixDiskInfo[:fileName]}"` in all cases?  It looks like it isn't dependent on the method of connection